<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDetailsWindow</class>
 <widget class="QDialog" name="OrderDetailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderDetailsWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Shipment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbName">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbSurname">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSurname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbCity">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCity"/>
     </item>
     <item>
      <widget class="QLabel" name="lbPostalcode">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Postal Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePostalCode"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbAddress">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lbCountry">
       <property name="minimumSize">
        <size>
         <width>46</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCountry"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Payment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bank username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBankUsername"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Bank password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBankPassword"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
